operations/setup.go

```diff
@@ -77,10 +77,19 @@ func (c *serviceConf) getSenders(conf *model.CedarConfig) (send.Sender, error) {
 			return nil, errors.Wrap(err, "problem building splunk logger")
 		}
 		if err = sender.SetErrorHandler(send.ErrorHandlerFromSender(fallback)); err != nil {
-			return nil, errors.Wrap(err, "problem configuring error handler")
+			return nil, errors.Wrap(err, "configuring splunk logger error handler")
 		}
-
-		senders = append(senders, send.NewBufferedSender(sender, loggingBufferDuration, loggingBufferCount))
+		opts := send.BufferedAsyncSenderOptions{}
+		opts.FlushInterval = loggingBufferDuration
+		opts.BufferSize = loggingBufferCount
+		bufferedSender, err := send.NewBufferedAsyncSender(ctx, sender, opts)
```

Comment:
Nit: can we combine this into one line with L82-84 above?

Reply:
done